### ISSUE TITLE
Add 'GeoJSON' as an option in the 'Select export type' drop down menu in the UI. 

### ISSUE THREAD

**Issue by mdestaubin** [kobotoolbox/kpi#2401]
Blocked by #2218.

## Description

A non-required, but a 'nice-to-have' feature as part of the UNEP work is to add GeoJSON as an export file type option in the KoBoToolbox interface.  Ideally, this would only be an option if there was at least one GPS question included in the form. 

## UI Scope
![Screenshot_091019_022404_PM](https://user-images.githubusercontent.com/31933880/64639809-a4089a00-d3d6-11e9-9f75-958add1218c9.jpg)

**Comment by joshuaberetta** [comment]
Closed by https://github.com/kobotoolbox/kpi/pull/2968
